napari/layers/shapes/_tests/test_shapes.py

```diff
@@ -600,6 +600,28 @@ def test_selecting_shapes():
     assert layer.selected_data == set()
 
 
+def test_removing_all_shapes_empty_list():
+    """Test removing all shapes with an empty list."""
```

Comment:
@sofroniewn , I didn't have a chance to try this out yet, but I took a quick look. I think it would be good to also check that the colors, properties, etc. are also reset in these tests.

Reply:
good idea and thanks for the quick look!! If'll have a chance to look in more detail in the future, maybe you can take over the PR and add that in too - there's no rush on this one. If not I can try too. thanks!!

Reply:
No problem. I can work on that this week.

Reply:
I've done this now and updated the PR to include the fix for #1730